app/assets/javascripts/discourse/app/components/user-card-contents.js

```diff
@@ -247,11 +256,6 @@ export default Component.extend(CardContentsBase, CanCheckEmails, CleansUp, {
       this._close();
     },
 
-    showUser(username) {
```

Comment:
Renaming this to `showUserAndClose` is a regression. There are plugins that override the template of this component and rely on `{{action "showUser"}}`

Reply:
Ah, this was a tricky case because this component contains both a method named `showUser` (coming from a mixin) _and_ a member of the `actions` hash with the same name. I think this highlights some of the potential confusion that can come from using an `actions` hash. But regardless, I didn't intend to introduce a regression.

I will restore `showUser` to the `actions` hash to avoid a breaking change, and perhaps rename `showUserAndClose` -> `handleShowUser`. And I'll review the rest of the PR again with this in mind.

Reply:
Finished this review and removed the regression.